### ISSUE TITLE
Error when importing account

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#10485]
### What happened?

tried to import account from farcaster for my existing profile that was bravely changed by API call

### Please reproduce in steps

hehe

### Expected Behavior

finished 

### Are you logged in?

None

### Screenshots

![image](https://github.com/kodadot/nft-gallery/assets/90852205/7c109651-33e1-424b-9b28-b50557ca6317)

### Wallet Address & Transaction hash

Address: 1cQB9WUTtwyH3H4Ksn7pLfJvMLCxtzQHbUNhZtVBasoX3X9

### Relevant log output

```shell
Url: https://canary.kodadot.xyz/ahp/u/1cQB9WUTtwyH3H4Ksn7pLfJvMLCxtzQHbUNhZtVBasoX3X9
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
Wallet: polkadot-js
Chain: ahp
Error: Error: [PROFILE::UPDATE] ERROR: Invalid type
```

### Payment link for reward

https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hassnian** [comment]
will be fixed by the time #10480 is merged

issue: request is missing `message` and `signature`, worker now requires it 


cc @Jarsen136

**Comment by Jarsen136** [comment]
> @Jarsen136 is the updated worker only deployed on beta or also prod ?
only on beta
